docs/interactive-api-client/README.md

```diff
@@ -0,0 +1,316 @@
+
```

Comment:
Can we put a comment at the top of this file which indicates it is generated and shouldn't be modified directly. I found this list of ways of adding comments to Markdown: https://stackoverflow.com/questions/4823468/comments-in-markdown
If the html style comments work that seems the best to me.